### PR TITLE
LWIMIQ post - euwe stabilization

### PR DESCRIPTION
**Pull request by bronaghs**

### CODE REVIEW

**Comment by JPrause** [comment]
@syncrou @imtayadeway @jntullo can you review this posting for LWIMIQ.
Thanks @bronaghs for your submission!!

File: site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Tired of having to input your RabbitMQ credentials on the rails console? Now you don't have to, thanks to [Miha](https://github.com/miha-plesko). Check out his work here: [Add GUI with inputs for VMware vCloud event monitoring](https://github.com/ManageIQ/manageiq/pull/11362).
+
+## Fixed
+[Šimon](https://github.com/isimluk) made a PR to fix the [Add a new chargeback rate screen](https://github.com/ManageIQ/manageiq/pull/12130) and 
+[Nick](https://github.com/carbonin) made a PR to [raise an error if a region is not configured for central admin](https://github.com/ManageIQ/manageiq/pull/12264).
+
+## New
+This week we saw the introduction of a new network provider page for Nuage, check out some nice eye candy [here](https://github.com/ManageIQ/manageiq/pull/10425) by [Samuel](https://github.com/samuel-nuage).
+
+Mac support was added to the [linux_block_device gem](https://github.com/ManageIQ/linux_block_device), thanks to [Jason](https://github.com/fryguy). No idea what the linux_block_device gem is? Me niether, have a [read](https://github.com/ManageIQ/linux_block_device).
```
Comment (jntullo) [inline site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md]: great line! 😂 there's a typo in "neither" though

**Comment by jntullo** [comment]
nice work @bronaghs 👍 😄

File: site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: Last week in ManageIQ - Euwe stabilization
+author: bronaghs
+date: 2016-10-31
+comments: true
+published: true
+tags: LWIMIQ
+---
+[Bronagh](https://github.com/bronaghs) here bringing you this weeks edition of Last Week in Manage IQ!
```
Comment (imtayadeway) [inline site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md]: I *think* this should be s/weeks/week's/ and s/Manage IQ/ManageIQ/

File: site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+and fixed a bunch of bugs while he was at it: [Rewrite of auto-tagging, fixing multiple bugs](https://github.com/ManageIQ/manageiq/pull/11806).
+
+Tired of having to input your RabbitMQ credentials on the rails console? Now you don't have to, thanks to [Miha](https://github.com/miha-plesko). Check out his work here: [Add GUI with inputs for VMware vCloud event monitoring](https://github.com/ManageIQ/manageiq/pull/11362).
+
+## Fixed
+[Šimon](https://github.com/isimluk) made a PR to fix the [Add a new chargeback rate screen](https://github.com/ManageIQ/manageiq/pull/12130) and 
+[Nick](https://github.com/carbonin) made a PR to [raise an error if a region is not configured for central admin](https://github.com/ManageIQ/manageiq/pull/12264).
+
+## New
+This week we saw the introduction of a new network provider page for Nuage, check out some nice eye candy [here](https://github.com/ManageIQ/manageiq/pull/10425) by [Samuel](https://github.com/samuel-nuage).
```
Comment (imtayadeway) [inline site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md]: Nit, but I think this needs more than a comma here. Perhaps dash or period?

File: site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+Mac support was added to the [linux_block_device gem](https://github.com/ManageIQ/linux_block_device), thanks to [Jason](https://github.com/fryguy). No idea what the linux_block_device gem is? Me neither, have a [read](https://github.com/ManageIQ/linux_block_device).
+
+VMware virtal machines can now be imported into RHEV using the REST API, [PR](https://github.com/ManageIQ/manageiq/pull/11455) by [Martin Beták](https://github.com/matobet).
+
+[Andrey](https://github.com/andyvesel) added a topology view his week to display the hosts on the backend of the OpenStack cloud, the PR can be found [here](https://github.com/ManageIQ/manageiq/pull/11679).
```
Comment (imtayadeway) [inline site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md]: s/his/this/

**Comment by imtayadeway** [comment]
Couple of minor things above, but looks great otherwise :+1:

**Comment by bronaghs** [comment]
Thank you! @jntullo @imtayadeway - Addressed.

File: site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+date: 2016-10-31
+comments: true
+published: true
+tags: LWIMIQ
+---
+[Bronagh](https://github.com/bronaghs) here bringing you this week's edition of Last Week in ManageIQ!
+## Featured
+A milestone was met this week when we entered into the stabilization phase of the upcoming Euwe release. At this point we will concentrate on fixing bugs. If you would like to help out then take your pick from our [open issues].
+
+Sprint 48 ended on Monday 24th. Don't worry if you could not make the Sprint review meeting this week, it was recorded [here](https://www.youtube.com/watch?v=wD6dqm9hSRM).
```
Comment (syncrou) [inline site/_posts/2016-10-28-lwimiq-euwe-stabilization.html.md]: Very nice!  👍

**Comment by syncrou** [comment]
👍 Looks great to me!

**Comment by JPrause** [comment]
@hayesr seems we have a quorum. Please merge.

**Comment by bronaghs** [comment]
Thanks everyone! ( such a supportive bunch 👍)

**Pull request merged**